README.md

```diff
@@ -405,6 +411,9 @@ onINP(addToQueue);
 onLCP(addToQueue);
 
 // Report all available metrics whenever the page is backgrounded or unloaded.
+// NOTE: do not use `document.addEventListener('visibilitychange'..)` since
```

Comment:
I think we need to investigate this more, and if it is a problem, fix the problem rather than recommending people not add this to `document`.

Reply:
In think I know what this is. 2 seconds...

Reply:
Removed this. Will discuss further in the issue: https://github.com/GoogleChrome/web-vitals/issues/502#issuecomment-2848090203